Resources/views/Collector/db.html.twig

```diff
@@ -411,3 +434,238 @@
 
     //]]></script>
 {% endblock %}
+
+{% block queries_v2 %}
```

Comment:
@javiereguiluz please take this comment into account. I won't merge it until it is done, and I will also delay the merge of the new design in Symfony until this is done, because I want to know how the upgrade path looks like for bundles before the feature is merged (we may need to change a bit the available CSS classes in the new design to make the upgrade path easier)